### PR TITLE
Legacy Convenience Image Deprecation #1561 

### PR DESCRIPTION
**Pull request by HussnainQuresshi**
CircleCI :  [Legacy Convenience Image Deprecation](https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034) are fixed as per new documentation.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -215,7 +215,7 @@ jobs:
   csv-lint:
     docker: # run the steps with Docker
       # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
-      - image: circleci/golang:1.13.3 #
+      - image: cimg/go:1.13.3 #
```
Comment (jbampton) [inline .circleci/config.yml:218]: Higher ?

File: .circleci/config.yml [hunk 2]
```diff
@@ -189,7 +189,7 @@ jobs:
   go-test:
     docker: # run the steps with Docker
       # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
-      - image: circleci/golang:1.13.3 #
+      - image: cimg/go:1.13.3 #
```
Comment (jbampton) [inline .circleci/config.yml:192]: Probably try Go 1.17

Comment (HussnainQuresshi) [reply .circleci/config.yml:192]: okey

File: .circleci/config.yml (hunk 1 above)
Comment (HussnainQuresshi) [reply .circleci/config.yml]: done

**Review by jbampton — APPROVED** [review]

**Pull request merged**